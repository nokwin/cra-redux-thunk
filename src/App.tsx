import React from "react";

function App() {
  return <div>nothing</div>;
}

export default App;
