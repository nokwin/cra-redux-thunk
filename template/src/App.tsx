import React from "react";
import { BrowserRouter, Route } from "react-router-dom";
import { Provider } from "react-redux";

import { HomePage } from "./pages/homepage/homepage.component";
import { CounterPage } from "./pages/counter-page/counter-page.component";
import { store } from "./redux/store";
import { Header } from "./components/header/header.component";
import { FormsPage } from "./pages/forms-page/forms-page.component";
import { ApiPage } from "./pages/api-page/api-page.component";

export const App = () => {
  return (
    <Provider store={store}>
      <BrowserRouter>
        <Header />
        <Route exact path="/" component={HomePage} />
        <Route exact path="/counter" component={CounterPage} />
        <Route exact path="/forms" component={FormsPage} />
        <Route exact path="/api" component={ApiPage} />
      </BrowserRouter>
    </Provider>
  );
};
