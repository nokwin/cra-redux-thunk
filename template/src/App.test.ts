import { App } from "./App";

describe("App component", () => {
  it("should render app component", () => {
    expect(App).toMatchSnapshot();
  });
});
