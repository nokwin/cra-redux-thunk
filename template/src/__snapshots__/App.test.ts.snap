// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component should render app component 1`] = `[Function]`;
